environment-api/src/test/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatusTest.java

```diff
@@ -72,6 +77,13 @@ void isAvailableTest(String testCaseName, EnvironmentStatus underTest, boolean a
                 { "UPDATE_FAILED",                          EnvironmentStatus.UPDATE_FAILED,                        true },
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS, false },
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS, false },
+                { "FREEIPA_DELETED_ON_PROVIDER_SIDE",       EnvironmentStatus.FREEIPA_DELETED_ON_PROVIDER_SIDE, true },
+                { "UPGRADE_CCM_VALIDATION_FAILED",          EnvironmentStatus.UPGRADE_CCM_VALIDATION_FAILED, true },
+                { "UPGRADE_CCM_ON_FREEIPA_FAILED",          EnvironmentStatus.UPGRADE_CCM_ON_FREEIPA_FAILED, true },
+                { "UPGRADE_CCM_ON_DATALAKE_FAILED",         EnvironmentStatus.UPGRADE_CCM_ON_DATALAKE_FAILED, true },
+                { "UPGRADE_CCM_ON_DATAHUB_FAILED",          EnvironmentStatus.UPGRADE_CCM_ON_DATAHUB_FAILED, true },
+                { "UPGRADE_CCM_FAILED",                     EnvironmentStatus.UPGRADE_CCM_FAILED, true },
+
```

Comment:
What about the new `*_IN_PROGRESS` and `UPGRADE_CCM_ROLLING_BACK` states?

Reply:
done